onnx/shape_inference/implementation.h

```diff
@@ -76,10 +105,47 @@ struct InferenceContextImpl : public InferenceContext {
     }
     return &allOutputTypes_[index];
   }
+
+  const GraphInferencer* getGraphAttributeInferer(
+      const std::string& attr_name) const override {
+    if (!graphInferenceContext_) {
+      fail_type_inference(
+          "GraphProto attribute inferencing is not enabled in this InferenceContextImpl instance.");
+    }
+
+    GraphInferencer* inferer = nullptr;
+
+    auto entry = graphAttributeInferers_.find(attr_name);
+    if (entry == graphAttributeInferers_.cend()) {
+      // create GraphInferencer instance
+      auto attrNameToGraphProto = graphProtoAttributesByName_.find(attr_name);
+      if (attrNameToGraphProto == graphProtoAttributesByName_.cend()) {
+        fail_type_inference(
+            "Attribute ", attr_name, " does not contain a graph.");
+      }
+
+      std::unique_ptr<GraphInferencer> new_inferer =
+          std::make_unique<GraphInfererImpl>(
```

Comment:
`std::make_unqiue` is available starting from c++14, while onnx needs to support c++11. Use the backports here instead https://github.com/onnx/onnx/blob/master/onnx/common/stl_backports.h (`ONNX_NAMESPACE::make_unqiue`).